src/lib/stores.js

```diff
@@ -0,0 +1,3 @@
+import { writable } from 'svelte/store';
+
+export const user = writable();
```

Comment:
This is going to be a global store that is shared across all users of the site, isn't it?

Reply:
Ugh, yes. I guess we should do something in regards to https://github.com/sveltejs/kit/discussions/4339